cmd/gw/main.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/spf13/cobra"
```

Comment:
can we change the name of the command from `gw` to `uplink` too?

Reply:
renamed the gw to uplink in the cmd/